p2p/pex/addrbook.go

```diff
@@ -222,6 +222,14 @@ func (a *addrBook) IsGood(addr *p2p.NetAddress) bool {
 	return a.addrLookup[addr.ID].isOld()
 }
 
+func (a *addrBook) IsBanned(addr *p2p.NetAddress) bool {
+	a.mtx.Lock()
+	defer a.mtx.Unlock()
+
```

Comment:
a.mtx.Lock()
_, ok := a.badPeers[addr.ID]
a.mtx.Unlock()

Reply:
can you explain to me the difference?

Reply:
no need in defer if you don't call other functions and code is not complex